src/api/routes.ts

```diff
@@ -348,7 +348,10 @@ export const buildItemPublishRoute = (itemId: UUID, notification?: boolean) =>
   )}`;
 
 export const buildPostEtherpadRoute = (parentId?: UUID) =>
-  `${ETHERPAD_ROUTE}/create${parentId ? qs.stringify({ parentId }) : ''}`;
+  `${ETHERPAD_ROUTE}/create${qs.stringify(
+    { parentId },
```

Comment:
What happens when the `parentId` is undefined ?

Reply:
qs should not prepend the query prefix